e2e/test/scenarios/dashboard-filters/dashboard-filters-nested.cy.spec.js

```diff
@@ -78,8 +78,11 @@ describe("scenarios > dashboard > filters > nested questions", () => {
     // Add multiple values (metabase#18113)
     filterWidget().click();
     popover().within(() => {
-      cy.findByText("Gizmo").click();
-      cy.findByText("Gadget").click();
+      cy.findByPlaceholderText("Enter some text")
```

Comment:
Previously we incorrectly assumed it's Products.Category field but it's coming from a SQL query without metadata and we shouldn't match it.